nservicebus/messaging/databus/azure-blob-storage.md

```diff
@@ -36,12 +36,12 @@ Review the [sample](/samples/azure/blob-storage-databus-cleanup-function/) to se
 
 ### Using the Blob Lifecycle Management policy
 
-Attachment blobs can be cleaned up using the [Blob Storage Lifecycle feature](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-lifecycle-management-concepts). This method allows configuring a single policy for all DataBus-related blobs. Those blobs can be either deleted or archived. The policy does not require custom code and is deployed directly to the storage account. This feature can only be used on GPv2 and Blob storage accounts, not on GPv1 accounts. 
+Attachment blobs can be cleaned up using the [Blob Storage Lifecycle feature](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-lifecycle-management-concepts). This method allows configuring a single policy for all data bus-related blobs. Those blobs can be either deleted or archived. The policy does not require custom code and is deployed directly to the storage account. This feature can only be used on GPv2 and Blob storage accounts, not on GPv1 accounts. 
 
 
 ## Configuration
 
-The following extension methods are available for changing the behavior of `AzureDataBus` defaults:
+The following extension methods are available for changing the behavior of `Azure Data Bus` defaults:
```

Comment:
`AzureDataBus` is the returned type (see the [code](https://docs.particular.net/nservicebus/messaging/databus/azure-blob-storage#configuration)) and should not be changed.
```suggestion
The following extension methods are available for changing the behavior of `AzureDataBus` defaults:
```